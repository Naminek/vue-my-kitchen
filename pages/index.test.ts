import Vuetify from 'vuetify'
import { shallowMount, createLocalVue } from '@vue/test-utils'
import IndexPage from './index.vue'

const localVue = createLocalVue()

describe('Dashboard page', () => {
  beforeEach(() => {
    jest.resetAllMocks()
  })

  function mountFunc (options = {}) {
    return shallowMount(IndexPage, {
      localVue,
      vuetify: new Vuetify(),
      ...options
    })
  }

  it('should match its snapshot', () => {
    const wrapper = mountFunc()
    expect(wrapper).toMatchSnapshot()
  })
})
