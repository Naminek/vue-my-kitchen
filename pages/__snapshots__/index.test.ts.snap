// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard page should match its snapshot 1`] = `
<v-row-stub align="center" class="mt-5 background-img" justify="center" tag="div">
  <v-col-stub cols="12" md="6" sm="8" tag="div">
    <v-card-stub loaderheight="4" tag="div">
      <v-card-title-stub class="headline" tag="div">
        Welcome to Namine's Kitchen!
      </v-card-title-stub>
      <v-card-text-stub tag="div">
        <p>You can check nice Japanese recipes based on categories, ingredients, and daily logs</p>
      </v-card-text-stub>
      <v-card-actions-stub tag="div">
        <v-spacer-stub tag="div"></v-spacer-stub>
        <v-btn-stub activeclass color="primary" nuxt="true" tag="button" to="/inspire" type="button">
          Explore
        </v-btn-stub>
      </v-card-actions-stub>
    </v-card-stub>
  </v-col-stub>
</v-row-stub>
`;
